pkg/cover/backend/elf.go

```diff
@@ -401,39 +402,104 @@ func symbolizeModule(target *targets.Target, objDir, srcDir, buildDir string,
 }
 
 // readCoverPoints finds all coverage points (calls of __sanitizer_cov_trace_*) in the object file.
-// Currently it is amd64-specific: looks for e8 opcode and correct offset.
+// Currently it is [amd64|arm64]-specific: looks for opcode and correct offset.
 // Running objdump on the whole object file is too slow.
-func readCoverPoints(file *elf.File, info *symbolInfo) ([2][]uint64, error) {
+func readCoverPoints(file *elf.File, info *symbolInfo, arch string) ([2][]uint64, error) {
+	var allPCs [2][]uint64
+	sections := []string{".text", ".init.text", ".exit.text"}
```

Comment:
Do we need ".init.text", ".exit.text"?
KCOV only collects coverage from syscalls issued by the fuzzer (from that particular task), we don't collect global coverage and we won't trace boot at all.
I understand that modules can be loaded and unloaded dynamically and as a result of syscalls as well.
But (1) this looks rather unlikely, (2) it's explicitly not supported by syzkaller (load addresses will change and coverage will explode, coverage reports won't work, (3) module init/exit code is constant and does not depend on syscall argument so there is nothing to fuzz (you can test it by loading and unloading the module once, which you hopefully already did, so there is no point in testing it more).
Am I missing something here?

Reply:
yeah, sound right, reverted that part.